webapp/backend/models/todos/todo.js

```diff
@@ -0,0 +1,13 @@
+// TODO: Convert code in this file over to NSMQ AI database models
```

Comment:
the sample code uses the React ToDo example. As the webapp work is being done these areas will be removed acordingly